docs/database/database-metrics.md

```diff
@@ -13,6 +13,8 @@ linkTitle: Metrics
 - [Database operation](#database-operation)
   - [Metric: `db.client.operation.duration`](#metric-dbclientoperationduration)
 - [Experimental](#experimental)
+  - [Database Response](#database-response)
```

Comment:
I think the distinction here was between database operations (like queries) vs low-level stuff like connection pool

So maybe 
```suggestion
  - [Database operations](#database-operations)
```

or just 

```suggestion
```